client/verta/verta/endpoint/_endpoint.py

```diff
@@ -88,8 +89,8 @@ def _date_updated(self, data):
         return data['date_updated']
 
     @classmethod
-    def _create(cls, conn, conf, workspace, public_within_org, path, description=None):
-        endpoint_json = cls._create_json(conn, workspace, public_within_org, path, description)
+    def _create(cls, conn, conf, workspace, path, description=None, public_within_org=None, visibility=None):
+        endpoint_json = cls._create_json(conn, workspace, path, description, public_within_org, visibility)
```

Comment:
Reordering the params so they make more sense. I double-checked that it's updated everywhere this is called (which isn't that many places)